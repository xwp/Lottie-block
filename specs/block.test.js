import {
   insertBlock,
   getEditedPostContent,
   createNewPost,
   activatePlugin,
   deactivatePlugin
 } from "@wordpress/e2e-test-utils";

jest.setTimeout(30000);

describe("Block", () => {
   it("Can add block", async () => {
     await createNewPost();
     await insertBlock("Lottie");
     expect(await getEditedPostContent()).toMatchSnapshot();
   });
 });

