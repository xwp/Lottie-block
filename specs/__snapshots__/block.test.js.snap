// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block Can add block 1`] = `
"<!-- wp:create-block/lottie -->
<div class=\\"wp-block-create-block-lottie\\"><lottie-player background=\\"transparent\\" src=\\"\\" autoplay loop style=\\"height:400px\\" speed=\\"1\\"></lottie-player></div>
<!-- /wp:create-block/lottie -->"
`;
